Source/ViewControllers/BrickCollectionView.swift

```diff
@@ -501,7 +509,10 @@ extension BrickCollectionView: AsynchronousResizableDelegate {
         weak var weakLayout = self.layout
         self.performBatchUpdates({
             weakLayout?.updateHeight(indexPath, newHeight: newHeight)
-            }, completion: completion)
+        }, completion: { completed in
+            let _ = self
```

Comment:
Isolated the change to just this line of code. If it's all good, accept the commit.